src/licensedcode/data/rules/gpl-2.0-plus_with_bison-exception-2.2_3.RULE

```diff
@@ -0,0 +1,37 @@
+  Distributed under the OSI-approved BSD License (the "License");
+  see accompanying file Copyright.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even the
+  implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the License for more information.
+
+   Copyright (C) 1984, 1989, 1990, 2000, 2001, 2002, 2003, 2004, 2005, 2006
```

Comment:
In most cases , we are trying to avoid keeping copyright statements in license detection rules. So IMHO you could remove these two lines

Reply:
This works perfectly. Pushing changes with removed lines.